build/tutorials/tutorials-contest/evoting-avalanche-dapp/README.md

```diff
@@ -0,0 +1,1133 @@
+    fuji: {
+      provider: function () {
+        return new HDWalletProvider({
+          mnemonic: MNEMONIC,
+          providerOrUrl: `https://api.avax-test.network/ext/bc/C/rpc`,
+        });
+      },
+      network_id: "*",
+      gas: 3000000,
+      gasPrice: 470000000000,
+      skipDryRun: true,
+    },
+  },
+  compilers: {
+    solc: {
+      version: "0.8.0"
+    }
+  }
+};
+```
+
+Note that we're setting the `gasPrice` and `gas` to the appropriate values for the Avalanche C-Chain. Here, you can see that we have used `contracts_build_directory` to change the default location of `artifacts` from the project root directory to the `src` folder. This is because React cannot access files that are present outside the `src` folder.
+
+### **Get Avalanche credentials**
+
+For deploying smart contracts we need two things: A node connected to the Avalanche network and an account with few AVAX. Avalanche connected node through RPC \(Remote Procedure Call\) is provided for free by the Avalanche Networks.
+
+Now we need an Avalanche wallet, where we would keep our funds, required for all the transactions on the network. So, visit [here](https://wallet.avax.network) and create an account. Save the mnemonic in a secure place \(we would need it later\). Instructions to add funds will be provided later in the tutorial.
+
+### **Add .env file**
+
+Now we need a **Avalanche** wallet, where we would keep our funds, required for all the transactions on the network. Visit the [Avalanche Wallet](https://wallet.avax.network/) and create an account. While creating your account you will see your public address and mnemonic. This public address will be required to transfer funds. Save the **mnemonic** in a secure place, we will need it later. Instructions to add funds will be provided later in the tutorial.
+
+Create a `.env` file in the project root folder. Please take a note that dot \(.\) is necessary for the `.env` file name. Now copy your Avalanche wallet's mnemonic in the .env file as shown below. In the .env file, **MNEMONIC** should be enclosed within double-quotes \(" "\).
+
+```bash
+MNEMONIC="<avalanche-wallet-mnemonic>"
+```
+
+> Never share or commit your `.env` file. It contains your credentials like `mnemonics`. Therefore, it is advised to add `.env` to your `.gitignore` file.
+
+Our project setup is now complete.
+
+To confirm that we are on the same page, run the following command in the project root folder.
+
+```bash
+npm start
+```
+
+It might take few seconds, to show output as in the image below.
+
+![](./assets/localhost-react-server.png)
+
+In a browser, visit the URL of our running dApp: [http://localhost:3000](http://localhost:3000). If you followed the above steps, you would see the page as shown below.
+
+![](./assets/localhost-frontend.png)
+
+## **Create Election smart contract**
+
+Create the file `Election.sol` \(`.sol` stands for Solidity\) inside of the `contracts` directory and paste the following code:
+
+```javascript
```

Comment:
could you please add the correct format type? So `solidity` instead of `javascript`. Please check other code blocks too.